templates/pioneer-citp-news-disinfo/measurements/measurements.1.schema.json

```diff
@@ -63,9 +74,20 @@
                     "domain": { "type": "string" },
                     "classification": {
                         "type": "string",
-                        "description": "deprecated, use pageClassification"
+                        "description": "deprecated, use classifierResults"
+                    },
+                    "pageClassification": {
+                        "type": "integer",
+                        "description": "deprecated, use classifierResults"
+                    },
+                    "classifierResults": {
+                        "type": "object",
+                        "properties": {
+                            "additionalProperties": {
+                                "type": "integer"
+                            }
```

Comment:
This will also need to be updated to mirror the other `classifierResults` object. It may be useful to add description for both fields, too.

Reply:
Oh gosh I forgot I had two, updating now